Real Analysis/real-analysis-notes.tex

```diff
@@ -573,4 +576,371 @@ \subsection{Measurable spaces}
     elements in $\Ep$.
   \end{enumerate}
 \end{defn}
-\end{document}
+\subsection*{(1/30/2017) Lecture 4}
+\subsection{Measures}
+Now that we have delved into a measurable space, we want to define a
+``measure'' on this space. In general, we want measures to behave
+similar to our intuition for how $n$-dimensional volume would
+behave. The definition of measure takes into account a few of these intuitions.
+\begin{defn}
+  Given a measurable space $(X,\M)$, a \de{measure} $\mu$ on this
+  measurable space is a function $\mu: \M \to [0,\infty]$ such that
+  \begin{enumerate}
+  \item $\mu(\emptyset) = 0$
+  \item ($\sigma$-additivity.) For all sequences $(E_n)_{n \geq 1}$ of
+    disjoint sets in $\M$, we get that \[
+      \mu(\bigcup_{n \geq 1} E_n) = \sum_{n=1}^\infty \mu(E_n)
+    \]
+  \end{enumerate}
+\end{defn}
+We note that the notation $[0,\infty]$ here is intentional. We can
+think of $\R = (-\infty,\infty)$ and $\ov{\R} = [-\infty,\infty]$
+where $\ov{R}$ is a metric space. In fact, we can define the metric
+$\phi: \ov{\R} \to [-1,1]$ as follows \[
+  x \mapsto
+  \begin{cases}
+    \frac{2}{\pi} \arctan x \text{ if } x \in \R \\
+    1 \text{ if } x = \infty \\
+    -1 \text{ if } x = -\infty
+  \end{cases}
+\]
+We can then put a subset topology on $\R \subset \ov{\R}$, but this is
+equivalent to the standard topology on $\R$.
+\begin{example}
+  Take $(a_n)_{n \geq 1}$ with $a_n \in \ov{R}$. Then, $\lim_{n \to \infty} a_n =
+  \infty$ means that, for all $A > 0$, there exists an $N > 0$ such
+  that, for all $n \geq N$, $a_n \geq A$ traditionally, but this same
+  definition would let the limit go to $\infty \in \ov{\R}$ with the
+  new structure.
+\end{example}
+After that digression, let us return to our discussion of measures.
+\begin{defn}
```

Comment:
maybe change from a a real def to something like, one would like to define this as ___, but this is undefined when uncountable, thus we must define it using the sup.